Gemfile

```diff
@@ -46,6 +55,16 @@ group :development do
   # Spring speeds up development by keeping your application running in the background. Read more: https://github.com/rails/spring
   gem 'spring'
   gem 'spring-watcher-listen', '~> 2.0.0'
+
+  gem 'letter_opener'
+  gem 'annotate'
+  gem "better_errors"
+  gem "binding_of_caller"
+
+  gem 'bullet'
+  gem 'rubocop'
+  gem 'paranoia', '~> 2.2'
```

Comment:
this needs to be moved to all group as this will be used in production for soft deleting records